cmd/preflight/cmd/helper.go

```diff
@@ -1,17 +1,32 @@
 package cmd
 
 import (
+	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/onsi/ginkgo/reporters/stenographer/support/go-colorable"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
 	"github.com/trilioData/tvk-plugins/internal"
+	"github.com/trilioData/tvk-plugins/tools/preflight"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"sigs.k8s.io/yaml"
 )
 
-func setupLogger(logFilePrefix string) error {
+type preflightCmdOps struct {
+	PreflightOps preflight.Options        `json:"preflightOptions"`
+	CleanupOps   preflight.CleanupOptions `json:"cleanupOptions"`
```

Comment:
The purpose of `preflightCmdOps` is to provide interface to run and cleanup commands.
If CommonOptions is added here then cmdOps struct instance must be passed down to the cleanup and run preflight checks funcs.

CommonOptions struct embedded in Run and Cleanup structs seems to be better structure.
@msurudwad please let me know your thoughts on this.

Reply:
If CommonOptions are kept in `preflightCmdOps` struct then, users won't need to specify same config twice in yaml file. At present, users will need to specify common values in both `run` and `cleanup` section. This can user experience can be enhanced.
what is the challenge in doing this `If CommonOptions is added here then cmdOps struct instance must be passed down to the cleanup and run preflight checks funcs`?

Reply:
This is an enhancement, no hard requirement to do so.

Reply:
Will take up this task in a separate ticket